src/components/PolaroidPicture/PolaroidPicture.js

```diff
@@ -0,0 +1,64 @@
+import React, { Component } from 'react';
+import {
+  PolaroidListItem,
+  PolaroidLink,
+  PolaroidStar,
+  PolaroidButton,
+  PolaroidLoading,
+} from './style.js'
+
+class PolaroidPicture extends Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      loading: true
+    };
+  }
+
+  render () {
+    const { gif, favorites, favoriteGif, isSingleView, getGif } = this.props;
+    if (!gif) {
+      return null;
+    }
+
+    const handleImageChange = () => {
+      this.setState({
+        loading: false
+      });
+    };
+
+    const isFavorite = favorites.includes(gif.id)
+    
+    return (
+      <PolaroidListItem isSingleView={isSingleView}>
+        <img src={gif.images.original.url} alt={gif.title} onLoad={handleImageChange.bind(this)} />
```

Comment:
binding this inside render is a bad practice, cause in each render it will create a new reference